ios/RNMagicScript/components/BaseNodes/Prism.swift

```diff
@@ -114,23 +149,54 @@ import SceneKit
         return self
     }
 
-    @objc func setDebugMode(_ debug: Bool) {
+    fileprivate func updateSize() {
+#if targetEnvironment(simulator)
+        debugNode?.scale = size
+#endif
+        editNode?.size = size
+    }
+    
+    @objc func updateDebugMode() {
+#if targetEnvironment(simulator)
```

Comment:
Because we don't want to see debug-lines in release mode, do we?
I've just realized that the macro should not check `targetEnvironment(simulator)` but rather something like `DEBUG` mode.
I leave current implementation as it is and we should introduce the DEBUG macro for the whole project.